src/anlz.rs

```diff
@@ -525,58 +525,58 @@ pub struct Phrase {
 
 /// Section content which differs depending on the section type.
 #[binrw]
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Eq)]
 #[br(import(header: Header))]
 pub enum Content {
     /// All beats in the track.
-    #[brw(pre_assert(header.kind == ContentKind::BeatGrid))]
+    #[br(pre_assert(header.kind == ContentKind::BeatGrid))]
```

Comment:
Does `pre_assert` only work with `br`? I used `brw` on purpose to prevent writing inconsistent data, but maybe it never worked anyway...